app/views/contexts/hub_context.py

```diff
@@ -102,19 +113,37 @@ def get_section_url(section_id, list_item_id) -> str:
 
         return url_for('questionnaire.get_section', section_id=section_id)
 
-    def _get_row_for_section(self, section_name, section_id, list_item_id=None):
+    def _get_row_for_repeating_section(self, section_title, section_id, list_item_id):
```

Comment:
`section_title` is unused

Reply:
Don't quite understand how that didn't throw a lint error.. 🤔